test/PanelSpec.js

```diff
@@ -278,37 +297,32 @@ describe('<Panel>', () => {
 
     it('Should be aria-expanded=true', () => {
       const instance = ReactTestUtils.renderIntoDocument(
-        <Panel collapsible expanded header="Heading">
+        <Panel collapsible>
+          <Panel.Heading>
+            <Panel.Title toggle>
+              Heading
+            </Panel.Title>
+          </Panel.Heading>
           Panel content
         </Panel>
       );
       const anchor = ReactDOM.findDOMNode(instance).querySelector('.panel-title a');
-      assert.equal(anchor.getAttribute('aria-expanded'), 'true');
+      assert.equal(anchor.className, 'collapsed');
```

Comment:
this shouldn't test for the className, the goal of this subset of tests is to make sure the correct `ARIA` roles and attributes are applied to the components. 

the title of the test is `"Should be aria-expanded=true"` so that should be the thing tested for.